src/components/page/Header/index.tsx

```diff
@@ -0,0 +1,65 @@
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { User } from '@onaio/session-reducer';
+import * as React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { Menu, Layout, Image } from 'antd';
+import { Link } from 'react-router-dom';
+import Logo from '../../../assets/images/opensrp-logo-color.png';
+import './Header.css';
+import { BACKEND_ACTIVE } from '../../../configs/env';
+import { BACKEND_LOGIN_URL } from '../../../constants';
+
+const SubMenu = Menu.SubMenu;
+
+/** interface for HeaderProps */
+export interface HeaderProps extends RouteComponentProps {
+  authenticated: boolean;
+  user: User;
+}
+
+/** default props for Header */
+const defaultHeaderProps: Partial<HeaderProps> = {
+  authenticated: false,
+  user: {
+    email: '',
+    name: '',
+    username: '',
+  },
+};
+
+/** The Header component */
+
+export const HeaderComponent = (props: HeaderProps): JSX.Element => {
```

Comment:
I think the more generally accepted way to write this type definition is something like this:

```ts
const Component: React.FC<Props> = (props) => (
```